projects/plugins/crm/admin/settings/partials/menu.block.php

```diff
@@ -91,7 +91,7 @@
 	$tabs['mail']          = __( 'Mail', 'zero-bs-crm' );
 	$tabs['maildelivery']  = __( 'Mail Delivery', 'zero-bs-crm' );
 	$tabs['mailtemplates'] = __( 'Mail Templates', 'zero-bs-crm' );
-	$tabs['oauth']         = __( 'OAuth Connections', 'zero-bs-crm' );
+	$tabs['oauth']         = __( 'OAuth Connection', 'zero-bs-crm' ); // phpcs:ignore WordPress.WP.GlobalVariablesOverride.Prohibited
```

Comment:
Heh, we definitely probably shouldn't have been doing this. I didn't realise `$tabs` was a WP global, but just confirmed.

Reply:
TIL here too